http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskServiceInterp.scala

```diff
@@ -254,7 +254,7 @@ object DiskServiceInterp {
     } yield PersistentDisk(
       DiskId(0),
       googleProject,
-      config.zone,
+      req.zone.getOrElse(config.zone),
```

Comment:
Wondering if we should change `zone` to `defaultZone` in `reference.conf`..

Reply:
will do